crypto/ocsp/test/s2n/OCSP-TEST.md

```diff
@@ -0,0 +1,138 @@
+## Overview
+This original OCSP-TEST.md file and CA, OCSP, and server certs were taken from s2n and modified for our needs.
+link: https://github.com/aws/s2n-tls/tree/main/tests/pems/ocsp
+The files in this directory represent a cert hierarchy to test OCSP response stapling. Contents of der responses are hard coded in the ocsp_test.cc test file.
+
+## CA
+- ca_cert.pem
+- ca_key.pem
+
+Issuer for all of the other certs in the directory.
+Since this is a test PKI, we do an intermediate for issuing leaf cert(s).
+
+## OCSP
+* ocsp_cert.pem
+* ocsp_key.pem
+
+Cert/key for the test OCSP responder. OCSP responses will be signed by the key.
+The CN for this cert matches the URI in the Server Cert's "Authority Information Access" x509 extension.
```

Comment:
Judging from this [description](https://knowledge.digicert.com/solution/SO7239.html), one should put the full URI in a CN, but it's often used incorrectly. The [RFC](https://datatracker.ietf.org/doc/html/rfc6125#section-6.4.4) also implies that a SAN field should be prioritized over the CN field too.  
I'd imagine that this would be for a web server or CA to handle, as they'd be the ones handing out certificates to use. OCSP uses the `CertID` to match certificates (OCSP acceptance [criteria](https://datatracker.ietf.org/doc/html/rfc6960#section-3.2)).